.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,35 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend] <Bug Name>"
+labels: kind/bug, area/backend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- See [an introduction of all options](https://www.kubeflow.org/docs/pipelines/installation/overview/). -->
+*  KFP version: <!-- See version number shows on bottom of KFP UI left sidenav. -->
+*  KFP SDK version: <!-- Please attach the output of this shell command: $pip list | grep kfp -->
+
+
+### Steps to reproduce
+
+<!--
+How can we make the problem occur?
```

Comment:
Suggested revision, please update all instances:

Specify how to reproduce the problem. This may include information such as: a description of the process, code snippets, log output, or screenshots.

Reply:
Sounds good! Done.